cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -231,6 +231,30 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         archetypes[0].delete();
         deleteByList(tags);
     });
+    it("Deletes assessments from an archived questionnaire from an archetype and application", function () {
+        //automates polarion MTA-441 and MTA-442
+        const applications = createMultipleApplications(1);
+        const archetypes = createMultipleArchetypes(1);
```

Comment:
if we're only creating one application and one archetype, why just not use the create directly, not createMultiple... method.

Reply:
Oh I didn't know that, thanks for sharing 👏🏽 
No what you did is better I think.

Reply:
NP!